src/enums/experience.ts

```diff
@@ -119,6 +119,13 @@ export const RegionsOperator = makeEnum({
 export type RegionsOperator =
   typeof RegionsOperator[keyof typeof RegionsOperator];
 
+/**
+ * Set of IAB signals that we can communicate
+ */
+export enum IABSignal {
+  GppUsp = 'GPP_USP',
```

Comment:
Interesting. I suppose this will be extended to include TCF, other GPP signals?

Reply:
I was figuring start with GPP, but I could definitely imagine refactoring TCF to include this as well, yeah!